cli/src/crate.ts

```diff
@@ -43,12 +43,16 @@ export default class Crate {
   }
 
   async finish(dylib: string) {
-    await remove(this.addon);
-    await copy(dylib, this.addon);
+    if (existsSync(this.addon)) {
```

Comment:
Being very pedantic, this has a race condition: another process could create the file in between the `existsSync` check and the `unlink`. Another way to do this would be to just call `await unlink(this.addon)` and catch and ignore an `ENOENT` error. It's an unlikely race condition, though, not a big deal.

Reply:
Oh actually now that we're depending on `rimraf` we can just use that here.